scrapers/Sayuncle.yml

```diff
@@ -0,0 +1,28 @@
+name: Sayuncle
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - sayuncle.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //h2[@class="sceneTitle"]
+      Date:
+        selector: //div[@class="col-6 sceneDate"]/text()
+        postProcess:
+          - parseDate: 01/02/2006
+      Performers:
+        Name: //a[@class="model-name-link"]
+      Tags:
+        Name: //div[@class="sceneCol sceneColCategories"]/a
```

Comment:
are there tags in any scene?

Reply:
Not at the moment. I've been using this scraper for a long time myself so I have no idea why tags are there.